tests/testthat/test-oc_clear_cache.R

```diff
@@ -5,8 +5,8 @@ test_that("oc_clear_cache clears cache", {
   # until a memoise >v.1.1 is released, we need to run oc_get_memoise() twice to
   # have it really cache results
   # https://github.com/ropensci/opencage/pull/87#issuecomment-573573183
```

Comment:
```suggestion
```
That bug only applied to the memoise/ratelimitr combo, IIRC. The `replicate()` on the next line can also go.